packages/web-components/src/components/cbp-banner/cbp-banner.specs.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Banner/Specifications" />
+
+# cbp-banner
+
+## Purpose
+
+The Banner is a container spanning the entire viewport meant to highlight urgent or important content.
+
+## Functional Requirements
+
+* Informational banners are the only supported variation of banner currently supported for web Components.
+  * These banners contain an icon as well as a heading followed by descriptive text.
+  * These banners contain a button to dismiss the banner.
+
+## Technical Specifications
+
+### User Interactions
+
+* Pressing the "Dismiss" button will hide banner from view (but not remove it from the DOM).
+* Clicking the backdrop or the designated close button will close (hide) the drawer.
+
+### Responsiveness
+
+* The banners are responsive and content will wrap and reflow down to phone viewport sizes.
+
+### Accessibility
+
+* Within your banner, use the appropriate heading level according to your document structure.
```

Comment:
I'd remove this line since the banner is before the h1 and no heading structure really makes sense.